src/components/Sections/Programs/tabs/Clips/List.js

```diff
@@ -25,13 +25,11 @@ const renderUnit = (unit, t) => {
 };
 
 const ClipsList = () =>
-  <div>
-    <UnitList
-      key="programs-clips"
-      namespace="programs-clips"
-      renderUnit={renderUnit}
-      extraFetchParams={{ content_type: CT_CLIP }}
-    />
-  </div>;
+  <UnitList
+    key="programs-clips"
+    namespace="programs-clips"
+    renderUnit={renderUnit}
+    extraFetchParams={{ content_type: CT_CLIP }}
+  />;
```

Comment:
Are you sure the wrapping < div > was not required for CSS?

Reply:
its a redundant div